packages/lesswrong/server/resolvers/forumEventResolvers.ts

```diff
@@ -22,11 +22,8 @@ addGraphQLResolvers({
       if (postIds?.length && !!delta) {
         const pointsPerPost = Math.abs(delta)
         postIds.forEach(postId => {
-          if (postId in voteData.points) {
-            voteData.points[postId] += pointsPerPost
-          } else {
-            voteData.points[postId] = pointsPerPost
-          }
+          // Each post gets points equal to the max change attributed to that post
+          voteData.points[postId] = Math.max(pointsPerPost, voteData.points?.[postId] ?? 0)
```

Comment:
Updated this based on Toby's response in [here](https://docs.google.com/document/d/1b94up2PvNNby7Os3gWReu4vV0OfDsJUSx9A9FEYPz9k/edit)